homeassistant/components/frontend/__init__.py

```diff
@@ -407,22 +406,25 @@ def __init__(self, use_repo):
     @asyncio.coroutine
     def get(self, request, extra=None):
         """Serve the index view."""
-        import hass_frontend
-
         hass = request.app['hass']
 
         if self.use_repo:
             core_url = '/home-assistant-polymer/build/core.js'
             compatibility_url = \
                 '/home-assistant-polymer/build/compatibility.js'
             ui_url = '/home-assistant-polymer/src/home-assistant.html'
+            icons_fp = ''
+            icons_url = '/static/mdi.html'
         else:
+            import hass_frontend
             core_url = '/static/core-{}.js'.format(
                 hass_frontend.FINGERPRINTS['core.js'])
             compatibility_url = '/static/compatibility-{}.js'.format(
                 hass_frontend.FINGERPRINTS['compatibility.js'])
             ui_url = '/static/frontend-{}.html'.format(
                 hass_frontend.FINGERPRINTS['frontend.html'])
+            icons_fp = '-{}'.format(hass_frontend.FINGERPRINTS['mdi.html'])
+            icons_url = '/static/mdi{}.html'.format(icons_fp)
```

Comment:
We can move this one back down because `icons-fp` will actually contain the dash, so the url is same for both?